File: Servidor/Transaccional/AccesoDatos/Dao/NHibernate/DaoServicioNHibernate.cs
using NLog;
using Trascend.Bolet.AccesoDatos.Contrato;
using Trascend.Bolet.ObjetosComunes.Entidades;

namespace Trascend.Bolet.AccesoDatos.Dao.NHibernate
{
    public class DaoServicioNHibernate : DaoBaseNHibernate<Servicio, string>, IDaoServicio
    {
        private static Logger logger = LogManager.GetCurrentClassLogger();
    }
}

File: src/Servidor/Transaccional/AccesoDatos_Fac/Dao/NHibernate/DaoServicioNHibernate.vb
Imports NLog
Imports Diginsoft.Bolet.AccesoDatos.Contrato
Imports Trascend.Bolet.ObjetosComunes.Entidades
Imports Trascend.Bolet.AccesoDatos.Dao.NHibernate
Namespace Dao.NHibernate
    Public Class DaoServicioNHibernate
        Inherits DaoBaseNHibernate(Of FacServicio, String)
        Implements IDaoServicio
        Private Shared logger As Logger = LogManager.GetCurrentClassLogger()
    End Class
End Namespace